aspnet/migration/rc1-to-rc2.rst

```diff
@@ -0,0 +1,352 @@
+
+.. code-block:: c# 
+
+  public Startup(IApplicationEnvironment applicationEnvironment)
+  {
+     var builder = new ConfigurationBuilder()
+       .SetBasePath(applicationEnvironment.ApplicationBasePath)
+  }
+
+To: 
+
+.. code-block:: c#
+
+  public Startup(IHostingEnvironment hostingEnvironment)
+  {
+     var builder = new ConfigurationBuilder()
+      .SetBasePath(hostingEnvironment.ContentRootPath)
+  }
+
+  
+Namespace and package id changes
+---------------------------------- 
+
+All Microsoft.AspNet.\* namespaces are renamed to Microsoft.AspNetCore.\*. 
+The EntityFramework.\* packages and namespaces are changing to Microsoft.EntityFrameworkCore.\*.
+All ASP.NET Core package versions are now 1.0.0-\*.
+
+Working with IIS
+----------------
+
+HttpPlatformModule
+^^^^^^^^^^^^^^^^^^
+
+HttpPlatformModule was replaced by ASP.NET Core Module. The web.config created by the Publish IIS tool now configures IIS to use ASP.NET Core Module instead of HttpPlatformHandler to reverse-proxy requests to Kestrel.
+
+The code snippet below shows how to configure the new Publish IIS tool in ``project.json`` file
+
+.. code-block:: Json 
+
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "1.0.0-*",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
+    }
+  },
+  "scripts": {
+    "postpublish": "dotnet publish-iis --publish-folder %publish:OutputPath% --framework %publish:FullTargetFramework%"
+  }
+
+The ASP.NET Core Module must be configured in ``web.config``:
+
+.. code-block:: Xml
+  
+  <configuration>
+    <system.webServer>
+      <handlers>
+        <add name="aspNetCore" path="*" verb="*" 
+		modules="AspNetCoreModule" resourceType="Unspecified"/>
+      </handlers>
+      <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" 
+	  stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" 
+	  forwardWindowsAuthToken="false"/>
+    </system.webServer>
+  </configuration>
+
+
+The Publish IIS tool 
+^^^^^^^^^^^^^^^^^^^^^
+
+The name of the package that contains the Publish IIS tool was changed to ``Microsoft.AspNetCore.Server.IISIntegration.Tools``. This requires changing your project.json file to inlude the ``Microsoft.AspNetCore.Server.IISIntegration.Tools`` package instead of the ``dotnet-publish-iis`` package.
+
+You can use the Publish IIS tool to publish your app with the ``web.config`` file that is required for your target environment. More information about Publish IIS changes is at https://github.com/aspnet/Announcements/issues/164.
+
+IIS integration middleware is now setup using ``WebHostBuilder`` in ``Program.Main``, and is no longer called in the ``Configure`` method of the ``Startup`` class.
+
+Json configuration syntax change 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``ConfigurationRoot.ReloadOnChanged`` is no longer available, add is added explicitly via ``ConfigurationBuilder.AddJsonFile``. Update ``launchSettings.json`` to remove web target and add the following:
```

Comment:
@danroth27 Yes I did mean `launchSettings.json`. This file needs to be updated since the profile for launching self hosted application has changed.

in RC1 it was :
```
    "web": {
      "commandName": "web",
      "environmentVariables": {
        "Hosting:Environment": "Development"
      }
    }
```
 This will no longer work and needs to be updated to what is listed here.

However, I do feel that this should be in a different section as it's not related to json configuration.